sourcegraph/sourcegraph
# search/coreworkflow: Show recent searches in autocompletion

This commit adds the feature to show recent searches in the
autocompletion input on the search homepage when the input is empty.



https://user-images.githubusercontent.com/179026/183091899-fc685dba-8e93-4adc-b244-9af54fc0ac5b.mp4





## Test plan

- Enable coreworkflow improvements
- Go to search homepage.
- Focusing the query input (without content) should show search history entries.

## App preview:

- [Web](https://sg-web-fkling-search-history.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-cceqybbdxx.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
